.github/workflows/ci.yaml

```diff
@@ -0,0 +1,26 @@
+name: "ci"
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      HF_TOKEN: ${{ secrets.HF_TOKEN }}
```

Comment:
I believe that the HF_TOKEN is required only because the tests are initialized with the tokenizer for llama / gemma. For the small tests it should be safe to just change it to random public tokenizer - eg. `gpt2`. This wouldn't require us to maintain a token with access to gated repositories